apollo-backend/src/main/java/io/logz/apollo/controllers/BlockerDefinitionController.java

```diff
@@ -106,6 +107,15 @@ public void addBlockerDefinition(String name, Integer environmentId, Integer ser
                 logger.error("Could not initiate a SingleRegionBlocker with environment or stack");
                 assignJsonResponseToReq(req, HttpStatus.BAD_REQUEST, "SingleRegionBlocker cannot be defined for a specific environment or stack");
                 return;
+            } else if (blockerJsonConfiguration.equals("{}")) {
+                Optional<String> configOpt = blockerService.initSingleRegionBlockerConfiguration(serviceId, stackId);
+                 if (configOpt.isPresent()) {
+                     blockerJsonConfiguration = configOpt.get();
+                 } else {
+                     logger.error("Could not initiate a SingleRegionBlocker without service or service-stack");
+                     assignJsonResponseToReq(req, HttpStatus.BAD_REQUEST, "SingleRegionBlocker cannot be defined without service(s)");
+                     return;
```

Comment:
addBlockerDefinition has 5 `return` statements. It is not a good practice, suggest to do some refactoring.

Reply:
@mikeskali What are you suggesting? Either way I have to use 5 if statement for to trigger a specific return message for the user.